### PR TITLE
Added Odigos in the list

### PR DESCRIPTION
**Pull request by Shorya18**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up
#801 
Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

**Comment by crweiner** [comment]
Hello @Shorya18! Can you please update this PR title to reflect the company you're looking to add to the Swag List, and verify that all looks as intended in the site preview: https://deploy-preview-805--hacktoberfest-swag-list.netlify.app/list/#odigos

There are some formatting issues I'd love for you to see and change. Be sure you're following the [Contributing guide](https://hacktoberfestswaglist.com/contributing/). Thanks!

**Comment by Shorya18** [comment]
> Hello @Shorya18! Can you please update this PR title to reflect the company you're looking to add to the Swag List, and verify that all looks as intended in the site preview: https://deploy-preview-805--hacktoberfest-swag-list.netlify.app/list/#odigos
> 
> There are some formatting issues I'd love for you to see and change. Be sure you're following the [Contributing guide](https://hacktoberfestswaglist.com/contributing/). Thanks!

Hey I have changed the tittle name to the company's name which I want to add in the list.
thanks

File: docs/list.md [hunk 1]
```diff
@@ -424,6 +424,14 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 
 ### O
 
+#### *Odigos*
+
+- **Swag**: Swag (Specific Swags not mentioned)
+- **Requirements**: Make 3 successful merged PRs by October 20,2023  to win swags
+- **How to sign up**: No special signup known.
+- **Issues**: All the [open issues](https://github.com/keyval-dev/odigos/issues)
+- **Notes**: [Full details are mentioned [here](https://github.com/keyval-dev/odigos/blob/main/README.md).
```
Comment (crweiner) [inline docs/list.md:433]:
```suggestion
- **Notes**: Full details are mentioned [here in the repo Readme](https://github.com/keyval-dev/odigos/blob/main/README.md).
```
The

File: docs/list.md [hunk 2]
```diff
@@ -424,6 +424,14 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 
 ### O
 
+#### *Odigos*
+
+- **Swag**: Swag (Specific Swags not mentioned)
```
Comment (crweiner) [inline docs/list.md:429]: The swag is mentioned in the[ Readme file](https://github.com/keyval-dev/odigos/blob/main/README.md), can you please update this?

File: docs/list.md [hunk 3]
```diff
@@ -424,6 +424,14 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+- **Requirements**: Make 3 successful merged PRs by October 20,2023  to win swags
```
Comment (crweiner) [inline docs/list.md:430]: Please watch your spacing here
```suggestion
- **Requirements**: Make 3 successful merged PRs by October 20, 2023
```

**Review by crweiner — CHANGES_REQUESTED** [review]
Hi @Shorya18, thank you for updating the title of this PR to match the organization you are looking to add! I would like to give you a chance to review the things I've pointed out as well. Please be sure to view the Netlify deployment and make sure that everything for Odigos looks as intended.


Please ping me when you'd like me to take another look, thanks!

**Comment by vimode** [comment]
Hey @crweiner I think some of these old PRs which are not being updated by the PR authors should be closed and corresponding issues be made open for others to add them to the list. It would be much simpler for you.

**Comment by crweiner** [comment]
Closed due to inactivity

**Pull request closed without merging**